package.json

```diff
@@ -1,14 +1,76 @@
 {
   "name": "linter-slim",
-  "linter-package": true,
-  "activationCommands": {},
-  "main": "./lib/init",
-  "version": "1.0.0",
+  "main": "./lib/linter-slim",
+  "version": "2.0.0",
   "description": "Lint Slim on the fly",
   "repository": "https://github.com/AtomLinter/linter-slim",
   "license": "MIT",
   "engines": {
-    "atom": ">0.50.0"
+    "atom": ">=1.9.0 <2.0.0"
   },
-  "dependencies": {}
+  "configSchema": {
+    "slimrbExecutablePath": {
```

Comment:
NOTE: If you want this to appear as "slimrb Executable Path" you'll need to add the `title` attribute to override the automatic "Slimrb Executable Path" that will be generated.